LICENSE

```diff
@@ -0,0 +1,191 @@
+any separate license agreement you may have executed with Licensor regarding
+such Contributions.
+
+6. Trademarks.
+
+This License does not grant permission to use the trade names, trademarks,
+service marks, or product names of the Licensor, except as required for
+reasonable and customary use in describing the origin of the Work and
+reproducing the content of the NOTICE file.
+
+7. Disclaimer of Warranty.
+
+Unless required by applicable law or agreed to in writing, Licensor provides the
+Work (and each Contributor provides its Contributions) on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied,
+including, without limitation, any warranties or conditions of TITLE,
+NON-INFRINGEMENT, MERCHANTABILITY, or FITNESS FOR A PARTICULAR PURPOSE. You are
+solely responsible for determining the appropriateness of using or
+redistributing the Work and assume any risks associated with Your exercise of
+permissions under this License.
+
+8. Limitation of Liability.
+
+In no event and under no legal theory, whether in tort (including negligence),
+contract, or otherwise, unless required by applicable law (such as deliberate
+and grossly negligent acts) or agreed to in writing, shall any Contributor be
+liable to You for damages, including any direct, indirect, special, incidental,
+or consequential damages of any character arising as a result of this License or
+out of the use or inability to use the Work (including but not limited to
+damages for loss of goodwill, work stoppage, computer failure or malfunction, or
+any and all other commercial damages or losses), even if such Contributor has
+been advised of the possibility of such damages.
+
+9. Accepting Warranty or Additional Liability.
+
+While redistributing the Work or Derivative Works thereof, You may choose to
+offer, and charge a fee for, acceptance of support, warranty, indemnity, or
+other liability obligations and/or rights consistent with this License. However,
+in accepting such obligations, You may act only on Your own behalf and on Your
+sole responsibility, not on behalf of any other Contributor, and only if You
+agree to indemnify, defend, and hold each Contributor harmless for any liability
+incurred by, or claims asserted against, such Contributor by reason of your
+accepting any such warranty or additional liability.
+
+END OF TERMS AND CONDITIONS
+
+APPENDIX: How to apply the Apache License to your work
+
+To apply the Apache License to your work, attach the following boilerplate
+notice, with the fields enclosed by brackets "[]" replaced with your own
+identifying information. (Don't include the brackets!) The text should be
+enclosed in the appropriate comment syntax for the file format. We also
+recommend that a file or class name and description of purpose be included on
+the same "printed page" as the copyright notice for easier identification within
+third-party archives.
+
+   Copyright [yyyy] [name of copyright owner]
```

Comment:
This line should be changed to
```
Copyright 2024 [C-Worthy]
```

Reply:
good catch, see https://github.com/CWorthy-ocean/C-Star/pull/34/files#r1834715608

Reply:
Actually no wait, if you read the context this part of the license is a set of instructions for how to add some text to other code, it's not something that needs to be replaced in this file.

That would also explain why xarray's version of this license also doesn't replace this line.

Reply:
I think what we are supposed to do here is leave the `LICENSE.txt` file unchanged, but copy this section with your suggested alteration and put it in the README

https://github.com/pydata/xarray/blob/a78f45a9c33b9fc9c10a69453e11722eb972cb92/README.md?plain=1#L113

Reply:
I don't agree. To me, it reads as if you are supposed to change this line. I think xarray just missed this.

Earthmover did this for icechunk as well:
https://github.com/earth-mover/icechunk/blob/main/LICENSE#L189

Reply:
> To apply the Apache License to your work, attach the following boilerplate
> notice, with the fields enclosed by brackets "[]" replaced with your own
> identifying information. (Don't include the brackets!)

Reply:
I can see how you can interpret this both ways. 
1. To only replace the brackets when you add this line **somewhere else**, or
2. To replace it also in the license so that other devs can simply copy from the license.

Reply:
> To me, it reads as if you are supposed to change this line.

To me this reads as "you should copy-paste this section, changing this line, and put it somewhere prominent elsewhere in your code (e.g. the readme)", not "change this line in-place here".

> Earthmover did this for icechunk as well:

That's weird - Icechunk doesn't "attach" the boilerplate notice anywhere. Somebody is doing it wrong 😆 

It seems that [Android itself](https://github.com/android/architecture-samples) just changes it in both places - I think we should follow that example.

Reply:
> Somebody is doing it wrong 😆
This sets the stage for a daring caper 🥷

Reply:
I've updated the PR to change it in both places